src/md/contributing.md

```diff
@@ -9,15 +9,48 @@ The apex core and apex malhar repositories both have mirror repositories on gith
 This project welcomes new contributors. If you would like to help by adding new features, enhancements or fixing bugs, here is how to do it.
 
 1. Create a [JIRA](https://malhar.atlassian.net/) for the work you plan to do (or assign yourself to an existing JIRA ticket)
-1. Fork the ASF github mirror (one time step):
-   https://github.com/apache/incubator-apex-core/
-1. Create a new branch from the [devel-3](https://github.com/apache/incubator-apex-core/tree/devel-3) branch. **Name your branch with the JIRA number in it, e.g. `APEX-123.my-feature`.**
-1. When adding new files, please include the Apache v2.0 license header.
+2. Fork the ASF github mirror (one time step):
```

Comment:
If we are viewing this in simple text editor, then it is confusing. Not everyone uses a markdown editor.